[editor]/editor_main/client/mapEditorScriptingExtension_c.lua

```diff
@@ -1,11 +1,6 @@
--- FILE: 	mapEditorScriptingExtension_c.lua
--- PURPOSE:	Prevent the map editor feature set being limited by what MTA can load from a map file by adding a script file to maps
--- VERSION:	RemoveWorldObjects (v1) AutoLOD (v1) BreakableObjects (v1)
-
-function requestLODsClient()
-	triggerServerEvent("requestLODsClient", resourceRoot)
-end
-addEventHandler("onClientResourceStart", resourceRoot, requestLODsClient)
+-- FILE: mapEditorScriptingExtension_c.lua
+-- PURPOSE: Prevent the map editor feature set being limited by what MTA can load from a map file by adding a script file to maps
+-- VERSION: RemoveWorldObjects (v1) AutoLOD (v1) BreakableObjects (v1)

```

Comment:
Probably worth bumping the version while at it, since something is being changed.
```suggestion
-- VERSION: RemoveWorldObjects (v1) AutoLOD (v2) BreakableObjects (v1)

```